docs/eventing/experimental-features.md

```diff
@@ -1,27 +1,35 @@
 # Experimental features
 
-In Knative we want to keep the innovation alive, experimenting and delivering new features without affecting the stability of the project.
+In Knative we want to keep the innovation alive, experimenting and delivering
+new features without affecting the stability of the project.
 
-In order to achieve that goal in Knative Eventing, we have a process to include new features.
-This allows users like you to try out new features and provide feedback back to the project.
+In order to achieve that goal in Knative Eventing, we have a process to include
+new features. This allows users like you to try out new features and provide
+feedback back to the project.
 
-This document explains how to enable experimental features and which ones are available today.
+This document explains how to enable experimental features and which ones are
+available today.
 
-For more details about the process, the feature phases, quality requirements and guarantees, check out the [Experimental features process documentation](https://github.com/knative/eventing/blob/main/docs/experimental-features.md).
+For more details about the process, the feature phases, quality requirements and
+guarantees, check out the
+[Experimental features process documentation](https://github.com/knative/eventing/blob/main/docs/experimental-features.md).
```

Comment:
My suggestion to replace this bit:

```
In order to keep Knative innovative, the maintainers of this project have developed an [experimental features process](https://github.com/knative/eventing/blob/main/docs/experimental-features.md), that allows new, experimental features to be delivered and tested by users, without affecting the stability of the core project.

<!--TODO: Add note about HOW / where users can provide feedback, otherwise there's not much point mentioning that-->

!!! warning
    Experimental features are unstable and may cause issues in your Knative setup or even your cluster setup. These features should be used with caution, and should never be tested on a production environment. For more information about quality guarantees for features at different stages of development, see the [Feature stage definition](https://github.com/knative/eventing/blob/main/docs/experimental-features.md#stage-definition) documentation.
```

Reply:
much better, thanks!

Reply:
You're very welcome! 😄